_includes/themes/cgal/default.html

```diff
@@ -131,7 +129,7 @@ <h1>{{ page.title }} {% if page.tagline %} <small>{{ page.tagline }}</small>{% e
             &copy; 1995-{{site.time | date: "%Y"}} The CGAL Project. All rights reserved.
             <div style="float: right;">
               <a href="https://cgal.geometryfactory.com/CGAL/Members/wiki/Main_Page" alt="Password protected access to the Developer Wiki">
-                <i class="glyphicon glyphicon-lock"></i>
+                <i class="bi bi-lock-fill fs-5"></i>
```

Comment:
You should search for all occurrences of `glyphicon` in the source code of the web site, to replace them.

Reply:
The remaining occurrences of `glyphicon` only refer to image names in `images/icons/` and are not related to the deleted font.

Reply:
In all those files, there are glyphicon:

```
_includes/themes/cgal/post.html
_posts/1998-04-01-cgal-10.md
_posts/1998-07-01-cgal-11.md
_posts/1999-01-01-cgal-12.md
_posts/1999-06-01-cgal-20.md
_posts/2000-01-01-cgal-21.md
_posts/2000-10-01-cgal-22.md
_posts/2001-10-01-cgal-23.md
_posts/2002-05-01-cgal-24.md
_posts/2003-11-01-cgal-30.md
_posts/2004-02-01-cgal-301.md
_posts/2004-12-01-cgal-31.md
_posts/2006-05-01-cgal-32.md
_posts/2006-07-01-cgal-321.md
_posts/2007-06-01-cgal-33.md
_posts/2007-09-04-cgal-331.md
_posts/2008-12-19-cgal-34-beta1.md
_posts/2009-01-22-cgal-34.md
_posts/2009-07-30-cgal-35-beta1.md
_posts/2009-10-05-cgal-35.md
_posts/2009-12-22-cgal-351.md
_posts/2010-03-22-cgal-36.md
_posts/2010-06-30-cgal-361.md
_posts/2010-08-23-cgal-37-beta.md
_posts/2010-10-15-cgal-37.md
_posts/2011-03-14-cgal-38-beta1.md
_posts/2011-04-19-cgal-38.md
_posts/2011-09-27-cgal-39.md
_posts/2012-03-12-cgal-40.md
_posts/2012-07-03-cgal-401.md
_posts/2012-07-04-cgal-402.md
_posts/2012-08-17-cgal-41-beta1.md
_posts/2012-10-24-cgal-41.md
_posts/2013-03-08-cgal-42-beta1.md
_posts/2013-04-11-cgal-42.md
_posts/2013-08-23-cgal-43-beta1.md
_posts/2013-10-17-cgal-43.md
_posts/2014-03-07-cgal-44-beta1.md
_posts/2014-04-04-cgal-44.md
_posts/2014-08-12-cgal-45-beta1.md
_posts/2014-10-10-cgal-45.md
_posts/2014-12-22-cgal-451.md
_posts/2015-02-15-cgal-452.md
_posts/2015-02-20-cgal-46-beta1.md
_posts/2015-04-19-cgal-46.md
_posts/2015-06-28-cgal-461.md
_posts/2015-08-07-cgal-47-beta1.md
_posts/2015-08-20-cgal-462.md
_posts/2015-09-21-cgal-463.md
_posts/2015-09-21-cgal-47-beta2.md
_posts/2015-10-20-cgal-47.md
_posts/2016-02-10-cgal48-beta1.md
_posts/2016-03-20-cgal48-beta2.md
_posts/2016-04-08-cgal48.md
_posts/2016-06-13-cgal-481.md
_posts/2016-08-04-cgal49-beta1.md
_posts/2016-09-19-cgal-482.md
_posts/2016-09-19-cgal-49.md
_posts/2017-04-12-cgal-491.md
_posts/2017-04-20-cgal410-beta1.md
_posts/2017-05-29-cgal410.md
_posts/2017-08-01-cgal411-beta1.md
_posts/2017-09-13-cgal-4101.md
_posts/2017-09-21-cgal411.md
_posts/2018-01-16-cgal-4102.md
_posts/2018-02-26-cgal4111.md
_posts/2018-02-27-cgal412-beta1.md
_posts/2018-04-09-cgal412-beta2.md
_posts/2018-04-25-cgal412.md
_posts/2018-07-10-cgal4112.md
_posts/2018-07-31-cgal4113.md
_posts/2018-08-01-cgal413-beta1.md
_posts/2018-09-04-cgal4121.md
_posts/2018-09-10-Periodic_Mesh_Generation.md
_posts/2018-09-11-cgal413-beta2.md
_posts/2018-10-01-cgal413.md
_posts/2019-01-23-Heat_Method.md
_posts/2019-01-29-VSA.md
_posts/2019-02-25-Hyperbolic_triangulations.md
_posts/2019-03-04-cgal414-beta1.md
_posts/2019-03-13-cgal414-beta2.md
_posts/2019-03-29-cgal414.md
_posts/2019-04-05-cgal4122.md
_posts/2019-04-05-cgal4131.md
_posts/2019-07-30-Shape_detection.md
_posts/2019-08-05-Polygonal_surface_reconstruction.md
_posts/2019-09-30-cgal4132.md
_posts/2019-09-30-cgal4141.md
_posts/2019-09-30-cgal50-beta1.md
_posts/2019-10-31-cgal50-beta2.md
_posts/2019-11-08-cgal4142.md
_posts/2019-11-08-cgal50.md
_posts/2020-01-24-cgal501.md
_posts/2020-02-25-cgal4143.md
_posts/2020-02-25-cgal502.md
_posts/2020-03-23-Registration.md
_posts/2020-04-02-Tutorial_on_Surface_Reconstruction.md
_posts/2020-04-20-Optimal_bounding_box.md
_posts/2020-05-04-Tutorial_on_GIS.md
_posts/2020-05-08-Surface_mesh_topology.md
_posts/2020-06-09-cgal51-beta1.md
_posts/2020-07-28-cgal51-beta2.md
_posts/2020-08-03-cgal503.md
_posts/2020-08-07-Tetrahedral-remeshing.md
_posts/2020-09-08-cgal51.md
_posts/2020-11-16-cgal504.md
_posts/2020-11-16-cgal511.md
_posts/2020-11-18-cgal52-beta1.md
_posts/2020-12-22-cgal512.md
_posts/2020-12-22-cgal52.md
_posts/2021-03-17-cgal513.md
_posts/2021-03-17-cgal521.md
_posts/2021-04-27-Orthtree.md
_posts/2021-06-02-cgal514.md
_posts/2021-06-02-cgal522.md
_posts/2021-06-04-cgal53-beta1.md
_posts/2021-06-21-Triangulations_on_the_sphere.md
_posts/2021-07-06-cgal515.md
_posts/2021-07-06-cgal523.md
_posts/2021-07-06-cgal53.md
_posts/2021-11-16-shape-regularization.md
_posts/2021-12-13-cgal524.md
_posts/2021-12-13-cgal531.md
_posts/2021-12-17-cgal54-beta1.md
_posts/2022-01-31-cgal54.md
_posts/2022-05-18-alpha_wrap.md
_posts/2022-06-06-cgal532.md
_posts/2022-06-06-cgal541.md
_posts/2022-06-06-cgal55-beta1.md
_posts/2022-07-15-cgal5.4.2.md
_posts/2022-07-15-cgal55.md
_posts/2022-10-12-cgal5.4.3.md
_posts/2022-10-12-cgal551.md
_posts/2023-02-28-cgal5.4.4.md
_posts/2023-05-09-improved_straight_skeleton.md
_posts/2023-02-28-cgal552.md
_posts/2023-06-13-cgal56-beta1.md
_posts/2023-07-28-cgal5.4.5.md
_posts/2023-07-28-cgal553.md
_posts/2023-07-28-cgal56.md
```

They all have exactly the same usage (because they are copy-pasted from each other):

```
_posts/2023-07-28-cgal56.md
10:<i class="glyphicon glyphicon-download"></i>
13:<i class="glyphicon glyphicon-book"></i>
```

plus:

```
_includes/themes/cgal/post.html
12:            <i class="glyphicon glyphicon-time"></i>
```

Those should be replaced using a script, to use the `bi`-equivalents.